installer/README.md

```diff
@@ -0,0 +1,49 @@
+# deno_install
+
+- This command installs executable deno script.
+
+## Features
+
+- Install executable script into ~/.deno/bin
+
+## Usage
+
+```sh
+$ deno_install https://deno.land/std/http/file_server.ts
+> file_server requests network access. Grant permanently? [yN] # Grant the permissions to use command.
+> y
+> Successfully installed file_server.
+$ file_server # now you can use installed command!
+```
+
+## Requirements
+
+- wget #TODO: Remove
+
+## Installing
+
+### 1. Install deno_install
+
+deno_install can be installed by using itself.
+
+```sh
+deno -A https://deno.land/std/install/deno_install.ts https://deno.land/std/install/deno_install.ts
+```
+
+### 2. Add `~/.deno/bin` to PATH
+
+```
+echo 'export PATH="$HOME/.deno/bin:$PATH"' >> ~/.bashrc # change this to your shell
```

Comment:
This message should be displayed to user after step 1. 

Also I think it makes sense to make it a param if message should be shown - if we add this script to deno as `deno install` then `.deno/bin` will already be in PATH. 

How about Windows?